server/migrations/1706065877825_update-feature-table.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined;
+
+exports.up = (pgm) => {
```

Comment:
This will work for what we need, since currently we don't have any data in these two tables.
Nit: I'd recommend dropping the constraint, instead of dropping the databases.

```
exports.up = (pgm) => {
  pgm.sql(`
  -- Drop the foreign key constraint for feature_id
  ALTER TABLE feature_to_login
  DROP CONSTRAINT IF EXISTS fk_feature_flag_feature_to_login;
  
  -- Drop the foreign key constraint for login_id
  ALTER TABLE feature_to_login
  DROP CONSTRAINT IF EXISTS fk_login_feature_to_login;
  `);
  pgm.sql(`  
  -- Drop the id column for feature_to_login table
  ALTER TABLE feature_to_login
  DROP COLUMN IF EXISTS id;

  -- Add the new id column
  ALTER TABLE feature_to_login 
  ADD COLUMN id integer GENERATED ALWAYS AS IDENTITY PRIMARY KEY;
  `);

  pgm.sql(`  
  -- Drop the id column for feature_flag table
  ALTER TABLE feature_flag
  DROP COLUMN IF EXISTS id;

  -- Add the new id column
  ALTER TABLE feature_flag 
  ADD COLUMN id integer GENERATED ALWAYS AS IDENTITY PRIMARY KEY;
  `);
};
```

Reply:
Saving this for future reference 🩷